packages/ckeditor5-dev-release-tools/tests/release/updateversions.js

```diff
@@ -0,0 +1,138 @@
+			expect( outputJsonSyncStub.callCount ).to.equal( 1 );
+			expect( outputJsonSyncStub.firstCall.args[ 0 ] ).to.contain( 'ckeditor5-dev' );
+			expect( outputJsonSyncStub.firstCall.args[ 1 ] ).to.deep.equal( { version: '1.0.1' } );
+		} );
+
+		it( 'should throw when npm show returns that version is already in use', () => {
+			getPackageJsonStub.returns( { version: '1.0.0', name: 'stub-package' } );
+			shExecStub.returns( '' );
+
+			expect( () => updateVersions( { version: '1.0.1' } ) )
+				.to.throw( Error, 'Provided version 1.0.1 is already used in npm by stub-package' );
+		} );
+
+		it( 'should not throw when npm show returns that version is not in use', () => {
+			shExecStub.throws( new Error( 'is not in this registry' ) );
+			getPackageJsonStub.returns( { version: '1.0.0', name: 'stub-package' } );
+
+			expect( () => updateVersions( { version: '1.0.1' } ) ).to.not.throw( Error );
+		} );
+
+		it( 'should throw when npm show throws another error', () => {
+			shExecStub.throws( new Error( 'custom error' ) );
+			getPackageJsonStub.returns( { version: '1.0.0', name: 'stub-package' } );
+
+			expect( () => updateVersions( { version: '1.0.1' } ) ).to.throw( Error );
+		} );
+
+		it( 'should call outputJsonSyncStub when version is 0.0.0-', () => {
+			updateVersions( { version: '0.0.0-nightly-123.0' } );
+
+			expect( outputJsonSyncStub ).to.be.called;
+		} );
+
+		it( 'should throw when new version is not greater than old version', () => {
+			gtStub.returns( false );
+
+			expect( () => updateVersions( { version: '1.0.0' } ) )
+				.to.throw( Error, 'Provided version 1.0.0 must be greater than 1.0.0.' );
+		} );
+
+		it( 'should throw when new version is not a valid semver version', () => {
+			validStub.returns( false );
+
+			expect( () => updateVersions( { version: '1.0.0' } ) )
+				.to.throw( Error, 'Provided version 1.0.0 must be a valid semver version.' );
+		} );
+
+		it( 'should pass cwd to globSync', () => {
+			updateVersions( { version: '1.0.1', cwd: 'Users/username/ckeditor5-dev/custom-dir' } );
+
+			expect( syncStub.firstCall.args[ 1 ] ).to.deep.equal( {
+				cwd: 'Users/username/ckeditor5-dev/custom-dir',
+				absolute: true,
+				nodir: true
+			} );
+		} );
+
+		it( 'should pass path with "package1" to readJsonSync to search for random packages when packagesDirectory is defined', () => {
+			syncStub.returns( [
+				'/ckeditor5-dev/packages/package1',
+				'/ckeditor5-dev'
+			] );
+
+			updateVersions( { version: '1.0.1', packagesDirectory: 'packages' } );
+
+			expect( readJsonSyncStub.firstCall.args[ 0 ] ).to.contain( 'package1' );
+		} );
+
+		it( 'should pass "/ckeditor5-dev" to readJsonSync to search for random packages when packagesDirectory is undefined', () => {
+			syncStub.returns( [ '/ckeditor5-dev' ] );
+
+			updateVersions( { version: '1.0.1' } );
+
+			expect( readJsonSyncStub.firstCall.args[ 0 ] ).to.equal( '/ckeditor5-dev' );
+		} );
```

Comment:
```suggestion
		it( 'should pass path with "package1" to readJsonSync to search for random packages when packagesDirectory is defined', () => {
			syncStub.returns( [
				'/ckeditor5-dev/packages/package1',
				'/ckeditor5-dev'
			] );

			updateVersions( { version: '1.0.1', packagesDirectory: 'packages' } );

			expect( readJsonSyncStub.firstCall.args[ 0 ] ).to.contain( 'package1' );
		} );

		it( 'should pass "/ckeditor5-dev" to readJsonSync to search for random packages when packagesDirectory is undefined', () => {
			syncStub.returns( [ '/ckeditor5-dev' ] );

			updateVersions( { version: '1.0.1' } );

			expect( readJsonSyncStub.firstCall.args[ 0 ] ).to.equal( '/ckeditor5-dev' );
		} );
```

Both tests should be merged with the proposal:

```
it( 'should not throw when the version is available (not in use)', () => {
```

The first one for the root package, the second one for the packages found in a packages directory.